### PR TITLE
Focus should be given to toast notifications

### PR DESCRIPTION
**Pull request by eleanor-heath**
https://github.com/adaptlearning/adapt-contrib-core/issues/51

### CODE REVIEW

File: js/views/notifyPopupView.js [hunk 1]
```diff
@@ -0,0 +1,266 @@
+import Adapt from 'core/js/adapt';
+import AdaptView from 'core/js/views/adaptView';
+
```
Comment (cahirodoherty-learningpool) [inline js/views/notifyPopupView.js:3]: Minor: I think we are trying to explicitly import wherever possible
```import Backbone from 'backbone';```

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

**Review by ethan-lp — APPROVED** [review]

**Review by lc-alexanderbenesch — APPROVED** [review]
👍

**Review by oliverfoster — CHANGES_REQUESTED** [review]
Linting warnings in notifyPopupView, notifyPushView and notifyView

Tested in Firefox, IE11, Chrome, with NVDA and JAWS. Reading perfectly.

**Review by oliverfoster — APPROVED** [review]

**Pull request merged**